src/app/components/hearing/events-and-audio/events-and-audio.component.ts

```diff
@@ -83,6 +85,16 @@ export class EventsAndAudioComponent implements OnInit, OnChanges, OnDestroy {
     }
   }
 
+  onPreviewAudio(id: number) {
+    if (!this.isAudioInPreview(id)) {
+      this.audioInPreview = [...this.audioInPreview, id];
```

Comment:
I think the scope of this ticket is to start the request to preview the audio, but not worry about the response.

Reply:
Ah, I just thought it was UI changes. I've handled the full thing in the next PR.

Reply:
That's fine, as discussed